samples/baremetal/v1/nodes/list_nodes.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+require 'vendor/autoload.php';
+
+$openstack = new OpenStack\OpenStack([
+    'authUrl' => '{authUrl}',
+    'region'  => '{region}',
+    'user'    => [
+        'id'       => '{userId}',
+        'password' => '{password}'
+    ],
+    'scope'   => ['project' => ['id' => '{projectId}']]
+]);
+
+$baremetal = $openstack->baremetalV1(['region' => '{region}']);
+
+$nodes = $baremetal->listNodes(['uuid' => '{uuid}']);
+
+foreach ($nodes as $node) {
+}
```

Comment:
mhhhh

Reply:
It's a code sample, only for documentation purpose. So I mainly copy pasted: https://github.com/php-opencloud/openstack/blob/master/samples/compute/v2/servers/list_servers.php